### PR TITLE
Support to show hub informations

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://github.com/stolostron/backlog/issues/25831
Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

**Comment by clyang82** [comment]
/assign @KevinFCormier Please take a look at. 

I will continue to increase the code coverage. Thanks.

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2242,6 +2243,12 @@
     "tab.overview": "Overview",
     "tab.submariner": "Submariner add-ons",
     "tab.userManagement": "User management",
+    "table.acm.channel": "Hub channel",
+    "table.acm.channel.helperText": "The channel that the subscription uses to install Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.consoleUrl": "Hub console URL",
+    "table.acm.consoleUrl.helperText": "The console URL of the Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.distribution": "Hub distribution version",
+    "table.acm.distribution.helperText": "The version of the Red Hat Advanced Cluster Management for Kubernetes.",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/translation.json:2251]: I think we need to remove the word "the" from before RHACM. @swopebe could you also review these, please?

Comment (swopebe) [reply frontend/public/locales/en/translation.json]: I think we can shorten this greatly with a little word change, goodness our product name is long!

**The Red Hat Advanced Cluster Management for Kubernetes console URL** 

**The Red Hat Advanced Cluster Management for Kubernetes version**

(depending on where this is, I don't think we need punctuation, but will let you guys figure that out @KevinFCormier -- it's not actually a complete sentence that warrants a period).

**Review by swopebe — APPROVED** [review]
Approved but with request to shorten the line with more concise language and revisiting punctuation strategy.

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (clyang82) [reply frontend/public/locales/en/translation.json]: Agree to follow our existing standard to keep consistence. Thanks @KevinFCormier and @swopebe 
"the" was removed in my morning time. so the PR is ready for review again.

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -2242,6 +2243,12 @@
     "tab.overview": "Overview",
     "tab.submariner": "Submariner add-ons",
     "tab.userManagement": "User management",
+    "table.acm.channel": "Hub channel",
+    "table.acm.channel.helperText": "The channel that the subscription uses to install Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.consoleUrl": "Hub console URL",
+    "table.acm.consoleUrl.helperText": "The console URL of Red Hat Advanced Cluster Management for Kubernetes.",
```
Comment (swopebe) [inline frontend/public/locales/en/translation.json:2249]: Ok, sure thing; not thrilled with this sentence structure but it is not wrong so approved on my end.

**Pull request merged**